### ISSUE TITLE
metronome function

### ISSUE THREAD

**Issue by dyj11** [jimutt/osmd-playback-demo#3]
Hi，
Have you considered implementing a metronome function?

**Comment by jimutt** [comment]
Yes, absolutely. But at the moment I'm not sure whether I will prioritize taking this prototype further and turning it into something more well polished and more feature-rich. So I'm afraid I can't make any promises about the future of this project.
